README.md

```diff
@@ -77,264 +81,298 @@ Community and partner projects:
+
+| Model            | Batch Size | Epochs | KNN Top1 Val Accuracy |
+|------------------|------------|--------|-----------------------|
+| BarlowTwins      |        256 |    800 |                 0.850 |
+| BYOL             |        256 |    800 |                 0.887 |
+| DCL              |        256 |    800 |                 0.864 |
+| DCLW             |        256 |    800 |                 0.861 |
+| DINO (Res18)     |        256 |    800 |                 0.887 |
+| FastSiam         |        256 |    800 |                 0.865 |
+| MAE (ViT-S)      |        256 |    800 |                 0.620 |
+| MSN (ViT-S)      |        256 |    800 |                 0.833 |
+| Moco             |        256 |    800 |                 0.874 |
+| NNCLR            |        256 |    800 |                 0.885 |
+| PMSN (ViT-S)     |        256 |    800 |                 0.830 |
+| SimCLR           |        256 |    800 |                 0.889 |
+| SimMIM (ViT-B32) |        256 |    800 |                 0.351 |
+| SimSiam          |        256 |    800 |                 0.871 |
+| SwaV             |        256 |    800 |                 0.899 |
+| SwaVQueue        |        256 |    800 |                 0.898 |
+| SMoG             |        256 |    800 |                 0.782 |
+| TiCo             |        256 |    800 |                 0.857 |
+| VICReg           |        256 |    800 |                 0.843 |
+| VICRegL          |        256 |    800 |                 0.781 |
+
+
+### Cifar10
+
+| Model         | Batch Size | Epochs | KNN Top1 Val Accuracy |
+|---------------|------------|--------|-----------------------|
+| BarlowTwins   |        512 |    800 |                 0.859 |
+| BYOL          |        512 |    800 |                 0.910 |
+| DCL           |        512 |    800 |                 0.874 |
+| DCLW          |        512 |    800 |                 0.871 |
+| DINO          |        512 |    800 |                 0.848 |
+| FastSiam      |        512 |    800 |                 0.902 |
+| Moco          |        512 |    800 |                 0.899 |
+| NNCLR         |        512 |    800 |                 0.892 |
+| SimCLR        |        512 |    800 |                 0.879 |
+| SimSiam       |        512 |    800 |                 0.904 |
+| SwaV          |        512 |    800 |                 0.884 |
+| SMoG          |        512 |    800 |                 0.800 |
+
 
 ## Terminology
 
-Below you can see a schematic overview of the different concepts present in the lightly Python package. The terms in bold are explained in more detail in our [documentation](https://docs.lightly.ai/self-supervised-learning/).
+Below you can see a schematic overview of the different concepts in the package.
+The terms in bold are explained in more detail in our [documentation](https://docs.lightly.ai/self-supervised-learning/).
 
 <img src="/docs/source/getting_started/images/lightly_overview.png" alt="Overview of the lightly pip package"/></a>
```

Comment:
This image is outdated :/ let's make a new one!

Reply:
Yes, we should also update it in the docs. Do you have the source?

Reply:
Updated the image :) The only major change is that I added a `Transforms` field. Let me know what you think.